File: python-playwright-enterprise-framework/utils/data_reader.py
import json
from pathlib import Path


ROOT_DIR = Path(__file__).resolve().parents[1]


def load_json(file_path):
    full_path = ROOT_DIR / file_path

    with open(full_path, "r", encoding="utf-8") as json_file:
        return json.load(json_file)

File: javascript-playwright-enterprise-framework/utils/data_reader.js
import fs from "node:fs";
import path from "node:path";
import { ROOT_DIR } from "./config_reader.js";

export function loadJson(filePath) {
  const fullPath = path.join(ROOT_DIR, filePath);
  const rawData = fs.readFileSync(fullPath, "utf-8");
  return JSON.parse(rawData);
}
